structopt-derive/src/parse.rs

```diff
@@ -39,6 +39,7 @@ pub enum StructOptAttr {
     // ident [= "string literal"]
     About(Ident, Option<LitStr>),
     Author(Ident, Option<LitStr>),
+    DefaultValue(Ident, Option<LitStr>),
```

Comment:
Default value can also be a global variable, not necessary a litstr.

Reply:
Or litstr name is misleading?

Reply:
I'm not sure I'm understanding you properly. 

This is just the parser, this code simply parses cases like 
```rust
#[structopt(default_value = "...")]
//                          ^^^^^  <-- this is represented as `LitStr` in syn
```

Cases like this will be handled by `NameLitStr` variant (i.e. just as regular raw method):
```rust
lazy_static! {
     static ref DEFAULT: &str = "...";
}

#[structopt(default_value = DEFAULT)]
//                          ^^^^^  <-- this is represented as `LitStr` in syn
```

By the way, this is exactly how `long`, `short`, `author` and the like work.

Reply:
OK, my bad.